xml_inscriptions/bib_only/MA.Bos.MFA.G.01.7479.xml

```diff
@@ -35,13 +35,78 @@
                </msIdentifier>
                <msContents>
                   <textLang mainLang="grc" otherLangs=""/>
-                  <msItem>
-                     <p/>
+                  <msItem class="#dedicatory #label">
+                     <p>dedication to Zeus on a fragment of a Corinthian-type helmet </p>
                   </msItem>
                </msContents>
-               <physDesc><!--objectDesc, handDesc, decoDesc go here--></physDesc>
-               <history><!--summary, origin, provenance go here--></history>
+               <!--  *************************************  <physDesc>  ********************************* -->
+               <physDesc>
+                  
+                  <!--  *************************************  <objectDesc>  *********************************  -->
+                  
+                  <objectDesc ana="#fragment #object_other">
+                     <supportDesc ana="#metal.bronze">
+                        <support>
+                           <p> bronze helmet </p>
+                           <dimensions type="surface" unit="cm">
+                              <height>25</height>
+                              <width>25</width>
+                              <depth/>
+                           </dimensions>
+                        </support>
+                        <condition ana="#complete.broken">
+                           <p> </p>
+                        </condition>
+                     </supportDesc>
+                     <!--  *************************************  <layoutDesc>  ********************************* -->
+                     <layoutDesc>
+                        <layout columns="0" writtenLines="1">
+                           <p/>
+                        </layout>
+                     </layoutDesc>
+                  </objectDesc>
+                  
+                  <!--  *************************************  <handDesc>  ********************************* -->
+                  
+                  <handDesc hands="1">
+                     <handNote ana="#inscribed"> "incised" </handNote>
```

Comment:
I'm not sure you need the "incised" here, as it's captured by the control vocabulary (this does make it sound to me that we might want to call this etched or something instead). Also, the inverted commas make it appear that you are taking it directly from someone else's description; if so, indicate where it's coming from. ("Per X, incised.", e.g.)